### PR TITLE
Update Beginner-friendly coding meetups

### PR DESCRIPTION
**Pull request by Nirvikalpa108**
I deleted a few meetups that haven't met in over a year and I updated Founders & Coders.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -71,9 +71,7 @@ <h2 id="technology">Technology-specific</h2>
...
-      <li><a href="https://www.facebook.com/groups/KeepCalmAndCarryOnCoding/" title="Keep Calm and Carry on Coding facebook group">Keep Calm and Carry on Coding</a> - a fortnightly meetup aimed at anyone have who has recently done a coding course and wants to continue learning.</li>
-      <li><a href="http://codeandcake.club">Code &amp;&amp; Cake</a> - regular meetups where you can work on your own coding side projects whilst eating cake (hurrah!); open to all experience levels.</li>
-      <li><a href="http://www.meetup.com/founderscoders/events/225965453/" title="Founders and Coders Coding for Everyone">Coding for Everyone (Founders &amp; Coders)</a> - find someone of a similar level to you to talk through and work on code with.</li>
+      <li><a href="https://www.meetup.com/founderscoders/" title="Founders and Coders Coding for Women">Coding for Women (Founders &amp; Coders)</a> - find someone of a similar level to you to talk through and work on code with.</li>
```
Comment (JennyBrennan) [inline index.html]: Suggest we take out the title attribute here - it doesn't add additional information for the user; sorry that the example you had to go on also had an unnecessary title attribute!

```suggestion
      <li><a href="https://www.meetup.com/founderscoders/">Coding for Women (Founders &amp; Coders)</a> - find someone of a similar level to you to talk through and work on code with.</li>
```

Comment (JennyBrennan) [reply index.html]: Based on the discussion in issue #8, could we replace line 83 with this Coding for Women link & description instead? Thank you! 👍

**Comment by Nirvikalpa108** [comment]
Great, I've accepted your changes (removed the title attribute and added the Coding for Women link and description). 

I hope I've done this correctly? Let me know if not! Thank you! 😃

**Comment by JennyBrennan** [comment]
Awesome - merging. Thanks so much for your work on this!

**Pull request merged**